### ISSUE TITLE
[1.2.5] TypeError when saving preferences

### ISSUE THREAD

**Issue by stefantalpalaru** [sharkwouter/minigalaxy#587]
Console output:

```text
$ minigalaxy 
src/nv_gbm.c:242: GBM-DRV error (nv_gbm_create_device_native): nv_common_gbm_create_device failed (ret=-1)

MESA-LOADER: failed to open nouveau: /usr/lib64/dri/nouveau_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib64/dri, suffix _dri)
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/minigalaxy/ui/preferences.py", line 202, in save_pressed
    DownloadManager.cancel_all_downloads()
TypeError: DownloadManager.cancel_all_downloads() missing 1 required positional argument: 'self'
```

This only appears when changing the installation path.

**Comment by sharkwouter** [comment]
Good find, thanks for the report. I'm looking into this.

**Comment by sharkwouter** [comment]
I managed to reproduce this and write a fix. I've added it to master. It should be in the next release.

**Comment by stefantalpalaru** [comment]
Fixed in 1.2.6.

**Comment by sharkwouter** [comment]
Thanks for updating this issue. Glad it works now.
